src/Reflection/ReflectionClassConstant.php

```diff
@@ -266,7 +267,8 @@ private function computeModifiers(ClassConst $node): void
         }
 
         if ($node->isPrivate()) {
-            $this->modifiers += CoreReflectionClassConstant::IS_PRIVATE;
+            // No += because private constant cannot be final
```

Comment:
BTW, I just realized that we made `computeModifiers()` an object that mutates `$this`, rather than something that produces a value (to then be assigned to `$this->modifiers`)

Reply:
Fixed.